app.go

```diff
@@ -105,6 +105,25 @@ func (a App) Groups(name string) (Group, error) {
 	return getResponse, nil
 }
 
+//All Groups...
```

Comment:
space 
```suggestion
// All Groups...
```

Reply:
Changed! Thanks, will close this PR and "reopen" it again on bitrise repo